TESTING.md

```diff
@@ -320,6 +320,28 @@ To open the context menu:
 4. Trigger autocomplete in the editor.
 5. Verify: A similar notification is shown to the user.
 
+## Context
```

Comment:
I think we should add link with checkbox to this header, just like the rest.

Reply:
Done